templates/generic.php

```diff
@@ -312,6 +364,260 @@
+                        <?=$this->e($pago20['NomBancoOrdExt'])?></span>
+                    <?php endif; ?>
+                    <?php if ('' !== $pago20['CtaOrdenante']) : ?>
+                        <span><strong>Cuenta ord:</strong> <?=$this->e($pago20['CtaOrdenante'])?></span>
+                    <?php endif; ?>
+                    <?php if ('' !== $pago20['RfcEmisorCtaBen']) : ?>
+                        <span><strong>RFC emisor cta ben:</strong> <?=$this->e($pago20['RfcEmisorCtaBen'])?></span>
+                    <?php endif; ?>
+                    <?php if ('' !== $pago20['CtaBeneficiario']) : ?>
+                        <span><strong>Cuenta ben:</strong> <?=$this->e($pago20['CtaBeneficiario'])?></span>
+                    <?php endif; ?>
+                    <?php if ('' !== $pago20['TipoCadPago']) : ?>
+                        <span><strong>Tipo cadena de pago:</strong> <?=$this->e($pago20['TipoCadPago'])?></span>
+                    <?php endif; ?>
+                </p>
+                <?php if ('' !== $pago20['CertPago']) : ?>
+                    <p>
+                        <strong>Certificado de pago:</strong>
+                        <span><?=$this->e($pago20['CertPago'])?></span>
+                    </p>
+                <?php endif; ?>
+                <?php if ('' !== $pago20['CadPago']) : ?>
+                    <p>
+                        <strong>Cadena de pago:</strong>
+                        <span><?=$this->e($pago20['CadPago'])?></span>
+                    </p>
+                <?php endif; ?>
+                <?php if ('' !== $pago20['SelloPago']) : ?>
+                    <p>
+                        <strong>Sello de pago:</strong>
+                        <span><?=$this->e($pago20['SelloPago'])?></span>
+                    </p>
+                <?php endif; ?>
+                <?php
+                $doctoRelacionados = $pago20->searchNodes('pago20:DoctoRelacionado');
+                if ($doctoRelacionados->count() > 0) :
+                    ?>
+                    <p style="margin: 10px 0 5px 0;">
+                        <strong>Documentos relacionados</strong>
+                    </p>
+                    <?php foreach ($doctoRelacionados as $doctoRelacionado) : ?>
+                    <p style="margin-bottom: 10px;">
+                        <strong>Id Documento: </strong><span><?=$this->e($doctoRelacionado['IdDocumento'])?></span>
+                        <strong>Serie: </strong><span><?=$this->e($doctoRelacionado['Serie'])?></span>
+                        <strong>Folio: </strong><span><?=$this->e($doctoRelacionado['Folio'])?></span>
+                        <strong>Moneda DR: </strong><span><?=$this->e($doctoRelacionado['MonedaDR'])?></span>
+                        <strong>Equivalencia DR: </strong>
+                            <span><?=$this->e($doctoRelacionado['EquivalenciaDR'])?></span>
+                        <strong>Tipo de cambio DR: </strong>
```

Comment:
Not needed, it is superseeded by `EquivalenciaDR`.

Reply:
Fixed at 2022-09-22